app/code/Magento/PageCache/etc/varnish6.vcl

```diff
@@ -62,43 +85,17 @@ sub vcl_recv {
         return (pass);
     }
 
-    # Bypass customer, shopping cart, checkout
-    if (req.url ~ "/customer" || req.url ~ "/checkout") {
-        return (pass);
-    }
-
```

Comment:
Could you please explain this change? Why you decided to remove this part?

Reply:
@peterjaap What was the deal again with this one? Did you say that Magento already returns `Cache-Control: private, no-cache, no-store` for these paths? Please clarify, I don't remember the exact reason.

Reply:
Yes exactly

Reply:
@ihor-sviziev is that an acceptable answer?

Long story short: no need to write code to bypass the cache if Magento prevents these pages from ending up in the cache.

Reply:
Make sense, thx

Reply:
@ThijsFeryn @peterjaap any updates?

Reply:
@peterjaap this is your domain, can you provide an answer to @ihor-sviziev please?

Reply:
@ihor-sviziev here:
- https://github.com/magento/magento2/blob/4a41ec4822424ccc579206ed2739527ccc9f49b7/lib/internal/Magento/Framework/App/Response/Http.php#L169
- https://github.com/magento/magento2/blob/4a41ec4822424ccc579206ed2739527ccc9f49b7/app/code/Magento/PageCache/Plugin/AppendNoStoreCacheHeader.php